client/renderers/EGL/desktop.c

```diff
@@ -35,6 +35,9 @@ Place, Suite 330, Boston, MA 02111-1307 USA
 #include "desktop_rgb.frag.h"
 #include "desktop_rgh.def.h"
```

Comment:
I just noticed the header is misnamed, `rgb` vs `rgh`

Reply:
The file is deleted for reasons mentioned in the other comment, no action is necessary.